to_report param

kodesiba opened the pull request:
Update axecute and log_write with to_report param, closes #32

bms63 reviewed (commented):
Do we want the add-in to have these options reflected in it?  Like a user could toggle these on/off with checkboxes?

kodesiba commented:
> Do we want the add-in to have these options reflected in it? Like a user could toggle these on/off with checkboxes?

Added these as a group of checkboxes that we can extend as we need other things, also added a waiter to the addin since I didn't know when the file was done when testing

bms63 commented:
Hey Nathan - I increased the size of the window so you don't have to scroll and put a fancy solar system spinner in there...that waiter package is really cool!

thomasp-b reviewed (approved):
Works like a charm!

nicholas-masel commented on R/timberAddin.R:
```diff
@@ -5,10 +5,12 @@
 #' @importFrom shiny textInput checkboxInput actionButton uiOutput reactiveValues observeEvent renderText stopApp runGadget conditionalPanel fluidRow column tags HTML
 #' @importFrom rstudioapi selectFile selectDirectory
 #' @importFrom stringr str_locate_all str_replace
+#' @importFrom waiter useWaiter waiter_show waiter_hide spin_fading_circles
```
```suggestion
#' @importFrom waiter useWaiter waiter_show waiter_hide spin_fading_circles spin_solar
```

bms63 replied:
Thanks Nick!
